src/SmartTransactionsController.ts

```diff
@@ -575,7 +569,6 @@ export default class SmartTransactionsController extends BaseController<
       this.updateSmartTransaction({
         chainId,
         nonceDetails,
-        metamaskNetworkId,
```

Comment:
Are we sure that in all places where `metamaskNetworkId` is referred to in the extension, chain ID is used instead?

Reply:
Uncertain how easy it will be to fully transition away from `metamaskNetworkId` for old transactions that predate `chainId`.

See https://github.com/MetaMask/metamask-extension/pull/20652#issuecomment-1707351348

Reply:
This is now resolved on extension side